ee/clickhouse/views/test/test_clickhouse_retention.py

```diff
@@ -88,17 +235,16 @@ def get_retention(client: Client, team_id: int, request: RetentionRequest):
     return client.get(
         f"/api/projects/{team_id}/insights/retention/",
         # NOTE: for get requests we need to JSON encode non-scalars
-        data={k: (v if isinstance(v, (str, numbers.Number)) else json.dumps(v)) for k, v in request.items()},
+        data={
+            k: (v if isinstance(v, (str, numbers.Number)) else json.dumps(v))
```

Comment:
json_encode_request_params?

Reply:
👍 will update

Reply:
have rebased and updated to use `json_encode_request_params`. I've also renamed to `encode_get_request_params` as this is the usage seems specific to this use case and includes omitting None